src/Uno.UI/Controls/CommandBar/CommandBarHelper.iOS.cs

```diff
@@ -61,34 +61,37 @@ public static void PageDestroyed(UIViewController pageController)
 		/// <param name="pageController">The controller of the page</param>
 		public static void PageWillAppear(UIViewController pageController)
 		{
-			var topCommandBar = pageController.FindTopCommandBar();
-			if (topCommandBar != null)
+			if (pageController.NavigationController is not null)
```

Comment:
Was this change supposed to be part of this PR?

Reply:
Yes and no, those are small changes part of a 17.4 preview changes because of nullability updates.